ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/validator/preexecution/WriteSetSelector.scala

```diff
@@ -3,9 +3,12 @@
 
 package com.daml.ledger.validator.preexecution
 
-/** Selects a write set from a [[PreExecutionOutput]]. */
+import com.daml.logging.LoggingContext
+
 trait WriteSetSelector[ReadSet, WriteSet] {
 
-  def selectWriteSet(preExecutionOutput: PreExecutionOutput[ReadSet, WriteSet]): WriteSet
+  def selectWriteSet(preExecutionOutput: PreExecutionOutput[ReadSet, WriteSet])(implicit
+      loggingContext: LoggingContext
+  ): WriteSet
```

Comment:
When given the choice, I prefer to put the explicit arguments on newlines rather than the implicit ones, because those are the ones I care about most of the time.

```suggestion
  def selectWriteSet(
      preExecutionOutput: PreExecutionOutput[ReadSet, WriteSet]
  )(implicit loggingContext: LoggingContext): WriteSet
```